swiss_locator/core/filters/swiss_locator_filter.py

```diff
@@ -384,7 +388,7 @@ def triggerResult(self, result: QgsLocatorResult):
             url_with_params = "&".join([f"{k}={v}" for (k, v) in params.items()])
 
             self.info(f"Loading layer: {url_with_params}")
-            wms_layer = QgsRasterLayer(url_with_params, result.displayString, "wms")
+            vt_layer = QgsRasterLayer(url_with_params, result.displayString, "wms")
```

Comment:
```suggestion
            ch_layer = QgsRasterLayer(url_with_params, result.displayString, "wms")
```
?

Reply:
Thanks, apparently the `IDE->Refactor->Rename` action went too far.